spm_BIDS.m

```diff
@@ -192,7 +192,7 @@
     if isempty(f), f = {}; else f = cellstr(f); end
     for i=1:numel(f)
         
-        p = parse_filename(f{i}, {'sub','ses','task','acq','rec','fa','echo','inv','run','recording', 'meta'});
+        p = parse_filename(f{i}, {'sub','ses','task','acq','ce','rec','dir','run','echo','recording','part','chunk','meta'});
```

Comment:
no part, chunk, echo or recording for events files (also note that this assumes that this is for func data)

https://bids-standard.github.io/bids-starter-kit/folders_and_files/files.html#func-functional-mri